packages/contracts/contracts/SmartInvoiceEscrow.sol

```diff
@@ -15,18 +15,18 @@ import {
 } from "@openzeppelin/contracts/proxy/utils/Initializable.sol";
 import {ISmartInvoiceEscrow} from "./interfaces/ISmartInvoiceEscrow.sol";
 import {ISmartInvoiceFactory} from "./interfaces/ISmartInvoiceFactory.sol";
-import {IArbitrable} from "@kleros/erc-792/contracts/IArbitrable.sol";
+import {
+    IDisputeResolver
+} from "@kleros/dispute-resolver-interface-contract/contracts/IDisputeResolver.sol";
 import {IArbitrator} from "@kleros/erc-792/contracts/IArbitrator.sol";
```

Comment:
We don't need this dependency. We can take IArbitrator from IDisputeResolver now